valmlord/gulpify
# Feature/rewrite code

## Description
This pull request updates the Gulp configuration to use ES6 syntax, renames tasks, and restructures the project files for better organization.

## Changes
- Updated all Gulp tasks to use ES6 syntax.
- Renamed existing Gulp tasks for clarity.
- Restructured the project files for improved organization.

## New Gulp Tasks
- Updated task names to reflect new conventions.
- All tasks now utilize ES6 features such as `import` and `export`.

## How to Test
1. Execute the Gulp tasks using the new task names.
2. Verify that the tasks run successfully and the project builds as expected.

## Additional Notes
- This update aims to modernize the Gulp setup by leveraging ES6 syntax and improving the overall project structure for better maintainability.
